magento/pwa-studio
# Cypress tag based testing

## Description

The changes will allow developers to run sub set of main suite based on Tags on CI and Tags/Strings on local. This helps developers not worry on entire suite but just changes related current work. 

## Related Issue







Closes [PWA-2699](https://jira.corp.magento.com/browse/PWA-2699)

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






1. `run cypress` should still work
2. `run cypress @ci` should run all tests tagged as @ci
3. `run cypress @invalidtag` should still pass but danger will throw warning about no tests run
## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
